lua/dylib.lua

```diff
@@ -0,0 +1,20 @@
+module(..., package.seeall)
+
+--
+-- C functions exported through FFI
+--
+
+local ffi = require("ffi")
+
+-- Structs
+ffi.cdef[[
+
+extern const char *CLIENT_PROC_ERROR;
+
+enum dyn_cmd_op {
```

Comment:
This ``enum`` defines the operations for different functional blocks, so that ``dy_cmd_proc()`` can use it to call the corresponding block's functions to deal with the client request.